lambdas/transcode/index.py

```diff
@@ -0,0 +1,54 @@
+"""
+Generate video previews for videos in S3.
+"""
+import subprocess
+import tempfile
+
+from t4_lambda_shared.decorator import api, validate
+from t4_lambda_shared.utils import get_default_origins, make_json_response
+
+
+SCHEMA = {
+    'type': 'object',
+    'properties': {
+        'url': {
+            'type': 'string'
+        }
+    },
+    'required': ['url'],
+    'additionalProperties': False
+}
+
+FFMPEG = '/opt/bin/ffmpeg'
+FFMPEG_TIME_LIMIT = 20
+DURATION = 5
+WIDTH = 320
+HEIGHT = 240
```

Comment:
Oh, sure. Actually, it could even be a query parameter - maybe the way the thumbnail lambda does it:
https://github.com/quiltdata/quilt/blob/master/lambdas/thumbnail/index.py#L36